doc/lua_api.md

```diff
@@ -7695,13 +7695,17 @@ child will follow movement and rotation of that bone.
 * `set_physics_override(override_table)`
     * Overrides the physics attributes of the player
     * `override_table` is a table with the following fields:
-        * `speed`: multiplier to default movement speed and acceleration values (default: `1`)
-        * `jump`: multiplier to default jump value (default: `1`)
-        * `gravity`: multiplier to default gravity value (default: `1`)
+        * `speed`: multiplier to *all* movement speed and acceleration values (default: `1`)
```

Comment:
To make this more concrete:

```suggestion
        * `speed`: multiplier to *all* movement speed (`speed_*`) and acceleration (`acceleration_*`) values (default: `1`)
```

In it's current form, one could think that it also means e.g. `liquid_sink`.